temporalcli/commands.server.go

```diff
@@ -144,11 +144,12 @@ func (t *TemporalServerStartDevCommand) run(cctx *CommandContext, args []string)
 		return err
 	}
 
-	cctx.Printer.Printlnf("%-16s %v:%v", "Temporal server:", toFriendlyIp(opts.FrontendIP), opts.FrontendPort)
+	cctx.Printer.Printlnf(VersionString() + "\n")
```

Comment:
```suggestion
	cctx.Printer.Printlnf("%v\n", VersionString())
```

This is technically safer if there are accidentally any printf components in the version string